discovery/testdata/single_provider_only_new_groups.yaml

```diff
@@ -0,0 +1,195 @@
+# No updates.
```

Comment:
We generally put these inline with the test code.

Reply:
I pushed https://github.com/prometheus/prometheus/pull/3426/commits/21e9505d19b0b15bc4a4118f0a0f4f31741a1270 to address this

Reply:
This branch is green again. ping @brian-brazil

Reply:
I meant that we don't have YAML files with Go objects, rather we have go objects inside the _test.go files.

Reply:
I started out with that approach but `_test.go` file became bloated with all the literal object declarations. You can see the example here : https://gist.githubusercontent.com/grandbora/bb4ef3af1588baa31e438054d438470f/raw/9d9d7eaca4bca739bed4e295d2e536e074a580ee/prometheus.tgs

It was really hard to work with that file afterwards. I can revert back to that but also if you have another suggestion I'd like to hear.

Reply:
This is what we do everywhere else, and we've far longer over in promql.

Reply:
Ok, I'll go back to that approach.

Reply:
Does this look good? https://github.com/prometheus/prometheus/pull/3426/commits/68344731087b9f7588ec17947f46a1f75a478a09